packages/cogify/src/cogify/gdal.command.ts

```diff
@@ -97,3 +97,31 @@ export function gdalBuildCog(targetTiff: URL, sourceVrt: URL, opt: CogifyCreatio
       .map(String),
   };
 }
+
+export function gdalCreate(targetTiff: URL, opt: CogifyCreationOptions): GdalCommand {
```

Comment:
can we doc what this function is trying to achieve, creating a empty background tiff to force a background for the tiff.